hack/ci-e2e-kind-ha-multi-zone.sh

```diff
@@ -11,6 +11,12 @@ set -x
 
 source $(dirname "${0}")/ci-common.sh
 
+# If running in prow, we need to ensure that garden.local.gardener.cloud resolves to localhost
+if [ -n "${CI:-}" -a -n "${ARTIFACTS:-}" ]; then
+    printf "\n127.0.0.1 garden.local.gardener.cloud\n" >> /etc/hosts
+    printf "\n::1 garden.local.gardener.cloud\n" >> /etc/hosts
+fi
```

Comment:
These lines have been added to all `ci-e2e-*.sh` files. Instead of copying it everywhere, please create a function in `hack/ci-common.sh` and call it here. The file is sourced in all the scripts anyway.